lib/Optimizer/GraphOptimizer/GraphOptimizer.cpp

```diff
@@ -2922,6 +2907,29 @@ bool OptimizeQuantization::run(Function *F, const CompilationContext &cctx) {
   return changed;
 }
 
+void glow::convertQuantizedConstants(Function *F, CompilationContext &cctx) {
+  for (auto &node : F->getNodes()) {
+    if (auto *Q = dyn_cast<QuantizeNode>(&node)) {
+      if (auto *C = dyn_cast<Constant>(Q->getInput())) {
+        // Quantize(Constant) -> Constant
+        // Note, it does not really matter how many usages this Constant has.
+        // Quantized graph will use optimized Constant and other functions will
+        // refer to the floating point original Constant.
+        NodeValue NC =
+            convertConstant(*F->getParent(), *C, Q->getResult().getType());
+        if (NC == NodeValue()) {
+          continue;
+        }
+        Q->getResult().replaceAllUsesOfWith(NC);
+        continue;
+      }
+    }
+  }
+
+  // Perform Dead Code Elimination.
+  runDCEPass(F, cctx);
```

Comment:
The optimization pipeline runs DCE before every pass (unless the pass opts out via `DCERequiredMode`, and none currently do). So not sure this is necessary.

Reply:
Done

Reply:
`convertQuantizedConstants` is not an optimization pass, but just a function called by `glow::optimizeFunction`. Then, DCE needs to be called manually.